extensions/wikia/Wall/WallThread.class.php

```diff
@@ -95,25 +95,20 @@ private function getReplyIdsFromDB( $dbr, $afterId = null ) {
 		// the other one is in Wall.class done in a grouped way
 		// (fetch for many threads at once, set with ->setReplies)
 
-		$conditions = [ 'parent_comment_id = '.$this->mThreadId ];
```

Comment:
Probably it would, but there is also ```'comment_id > '.$afterId``` which adds complexity in escaping for this function.